docker/perf_layer_env.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Enable the performance layers by setting the evironment variables.
+# Run vkcube and make sure it runs without fault when performance layers are 
+# enabled.
+# Test if each layer write to its corresponding output file. 
+
+set -e
+
+# Create a temp output directory for log files. 
+readonly OUTPUT_DIR="$(mktemp -d)"
+readonly INSTALL_DIR="$1"
+readonly LAYER_DIR="$2"
+
+declare -a output_files
+output_files=("compile_time.csv" "run_time.csv" "memory_usage.csv" "frame_time.csv")
+
+#######################################
+# Check if the layers write data in their specified log files.
```

Comment:
Excellent, this documentation is very useful and clear!

nit: `s/Check/Checks/`